addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5826,10 +5859,21 @@ registry.SnippetMove = SnippetOptionWidget.extend(ColumnLayoutMixin, {
     _swapMobileOrders(widgetValue, siblingEls) {
         const targetMobileOrder = this._getItemMobileOrder(this.$target[0]);
         const orderModifier = widgetValue === "prev" ? -1 : 1;
-        const newOrderClass = `order-${parseInt(targetMobileOrder[1]) + orderModifier}`;
-        const comparedEl = [...siblingEls].find(el => el.classList.contains(newOrderClass));
-        this.$target[0].classList.replace(targetMobileOrder[0], newOrderClass);
-        comparedEl.classList.replace(newOrderClass, targetMobileOrder[0]);
+        let delta = 0;
+        while (true) {
+            delta += orderModifier;
+            const newOrderClass = `order-${parseInt(targetMobileOrder[1]) + delta}`;
+            const comparedEl = [...siblingEls].find(el => el.classList.contains(newOrderClass));
+            if (!comparedEl) {
+                break;
+            }
```

Comment:
Is this necessary? You shouldn't be able to swap the orders with a non-existing sibling in the first place (i.e. if there is no arrow to reorder, you can't reorder).

Reply:
In master I agree, but in stable no caller of this method expects it to be able to raise an exception. Not having it would be equivalent to a change of signature IMO.